ern-container-gen/src/copyRnpmAssets.ts

```diff
@@ -24,7 +24,7 @@ export function copyRnpmAssets(
 
   for (const miniAppPath of miniAppPaths) {
     const packageJson = readPackageJsonSync(miniAppPath)
-    if (packageJson.rnpm && packageJson.rnpm.assets) {
+    if (packageJson?.rnpm?.assets) {
```

Comment:
So `packageJson` can be null? Previously it was a potential bug?

Reply:
No you are right, it shouldn't be `packageJson?`, will update it.